src/test/java/pl/damianszczepanik/jenkins/buildhistorymanager/utils/JobBuilder.java

```diff
@@ -17,7 +17,7 @@ public static Job buildSampleJob() {
 
     public static Job buildSampleJob(Run lastBuild) {
         Job job = mock(Job.class);
-
+        when(job.getFullName()).thenReturn("sampleJob");
```

Comment:
why if you don't validate it

Reply:
if nothing is configured `getFullName()` returns null which will crash

Reply:
where?

Reply:
On concatinating the log output.
Besides that in the "real world" this is never null